### PR TITLE
local socket binding for TCPConnector

### PR DESCRIPTION
**Pull request by gagoman**
As discussed here:

https://groups.google.com/forum/#!topic/aio-libs/lM7IUZq3DNM

Please list tests that should be written.

### CODE REVIEW

File: docs/client_reference.rst [hunk 1]
```diff
@@ -697,6 +697,9 @@ TCPConnector
       *ssl_context* may be used for configuring certification
       authority channel, supported SSL options etc.
 
+  :param tuple local_addr: tuple of ``(local_addr, local_port)`` used to bind
```
Comment (asvetlov) [inline docs/client_reference.rst]: Don't use markup here but update ./docs/client_reference.rst also.
`(local_addr, local_port)` should be `(local_host, local_port)` pair.

**Comment by asvetlov** [comment]
I suggest the following test scenario:
1. Select unused port on loopback (127.0.0.1) interface.
2. Make http request with passing selected local_addr
3. Check that socket from `connector._conns` transport has local_addr as `sock.gethostname()`

**Comment by asvetlov** [comment]
Is it possible to use `(0.0.0.0, 1234)` and `(127.0.0.1, 0)` pairs for `local_addr`?

**Comment by akhomchenko** [comment]
@asvetlov 

As for:

> (0.0.0.0, 1234)

`0.0.0.0` was resolved to `127.0.0.1` in my case

> (127.0.0.1, 0)

`0` port is translated to first free port, so it would not be `(127.0.0.1, 0)`

**Comment by akhomchenko** [comment]
@asvetlov looks like builds for `Python 3.5` are broken

**Comment by asvetlov** [comment]
Thanks!

**Pull request merged**